kalite/caching/tests/cache_tests.py

```diff
@@ -49,30 +49,30 @@ def test_cache_across_clients(self):
+        # Get a random content id
+        n_contents = len(self.content_cache)
+        content_id = random.choice(self.content_cache.keys())
+        logging.debug("Testing on content_id = %s" % content_id)
+        content_path = self.content_cache[content_id]['path']
 
         # Clean the cache for this item
-        caching.expire_page(path=video_path, failure_ok=True)
-        self.assertTrue(not caching.has_cache_key(path=video_path), "expect: No cache key after expiring the page")
+        caching.expire_page(path=content_path, failure_ok=True)
+        self.assertTrue(not caching.has_cache_key(path=content_path), "expect: No cache key after expiring the page")
 
         # Set up the cache with Django client
-        Client().get(video_path)
-        self.assertTrue(caching.has_cache_key(path=video_path), "expect: Cache key exists after Django Client get")
-        caching.expire_page(path=video_path) # clean cache
-        self.assertTrue(not caching.has_cache_key(path=video_path), "expect: No cache key after expiring the page")
+        Client().get(content_path)
+        self.assertTrue(caching.has_cache_key(path=content_path), "expect: Cache key exists after Django Client get")
+        caching.expire_page(path=content_path) # clean cache
+        self.assertTrue(not caching.has_cache_key(path=content_path), "expect: No cache key after expiring the page")
 
         # Get the same cache key when getting with urllib, and make sure the cache is created again
-        urllib.urlopen(self.live_server_url + video_path).close()
-        self.assertTrue(caching.has_cache_key(path=video_path), "expect: Cache key exists after urllib get")
-        caching.expire_page(path=video_path) # clean cache
-        self.assertTrue(not caching.has_cache_key(path=video_path), "expect: No cache key after expiring the page")
+        urllib.urlopen(self.live_server_url + content_path).close()
+        self.assertTrue(caching.has_cache_key(path=content_path), "expect: Cache key exists after urllib get")
+        caching.expire_page(path=content_path) # clean cache
+        self.assertTrue(not caching.has_cache_key(path=content_path), "expect: No cache key after expiring the page")
 
         # Same deal, now using requests library
-        requests.get(self.live_server_url + video_path)
-        self.assertTrue(caching.has_cache_key(path=video_path), "expect: Cache key exists after requestsget")
-        caching.expire_page(path=video_path) # clean cache
-        self.assertTrue(not caching.has_cache_key(path=video_path), "expect: No cache key after expiring the page")
+        requests.get(self.live_server_url + content_path)
+        self.assertTrue(caching.has_cache_key(path=content_path), "expect: Cache key exists after requestsget")
+        caching.expire_page(path=content_path) # clean cache
+        self.assertTrue(not caching.has_cache_key(path=content_path), "expect: No cache key after expiring the page")
```

Comment:
We should probably burn down the page cache stuff as well. I think it was only being used on the homepage and on topic pages (in order to mitigate the fact that these pages were querying the disk for availability, and hence slow). Now, with the sidebar topic tree nav as a single-page app, we won't be leveraging (or needing) any of that.

Any pages we can think of that *would* currently benefit from caching?